portfolio/pom.xml

```diff
@@ -27,6 +27,20 @@
       <version>4.0.1</version>
     </dependency>
 
+    <!--DataStore Library-->
+    <dependency>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>google-cloud-datastore</artifactId>
+        <version>1.104.0</version>
+    </dependency>
+
+    <!-- jsoup -->
+    <dependency>
+      <groupId>org.jsoup</groupId>
+      <artifactId>jsoup</artifactId>
+      <version>1.8.3</version>
+    </dependency>
```

Comment:
ohh what's this for>

Reply:
I'm use for erase html o java code that users submit in the forms